tetris.sh

```diff
@@ -17,13 +17,60 @@
 #
 # Author: Kirill Timofeev <kt97679@gmail.com>
 #
+# Localized Support: Rojen Zaman <rojen@riseup.net> | lang/README.md
+#
 # This program is free software. It comes without any warranty, to the extent
 # permitted by applicable law. You can redistribute it and/or modify it under
 # the terms of the Do What The Fuck You Want To Public License, Version 2, as
 # published by Sam Hocevar. See http://www.wtfpl.net/ for more details.
 
 set -u # non initialized variable is an error
 
+# show help page when given -h argument
+function help_page() {
+cat <<EOF
+usage: ./`basename $0` -l <language>
+supported languages:
+english (it is defualt, option not required)
+$(ls -I README.md -1t lang/ | cut -d'.' -f1)    
+EOF
+exit 0;
+}
+
+# BEGIN OF LANGUAGE LINES
+# if you want see or add other languages, visit lang/ dir and read lang/README.md
+# default localized values english:
+local_LinesComplated="Lines completed: ";
+local_Level="Level:           ";
+local_Score="Score:           ";
+local_UseCursorKeys="  Use cursor keys";
+local_Or="       or";
+local_Rotate="    s: rotate";
+local_LeftRight="a: left,  d: right";
+local_Drop="    space: drop";
+local_Quit="      q: quit";
+local_ToggleColor="  c: toggle color";
+local_ToggleShowNext="n: toggle show next";
+local_ToggleThisHelp="h: toggle this help";
+local_GameOver="Game over!";
+
+while getopts ":l:h" opt; do
+  case ${opt} in
+    l )
+        langChocie=${OPTARG};
+        . lang/$langChocie.sh &> /dev/null
+    ;;
+    h )
+        help_page;
+    ;;
+    : )
+        echo "Missing option argument for -$OPTARG" # if value of -l not given
```

Comment:
I think we should not only print error but also show usage, what do you think?

Reply:
okay, I will add usage command to end of error message